src/frontend/src/eth/providers/alchemy.providers.ts

```diff
@@ -83,6 +83,15 @@ export const initPendingTransactionsListener = ({
 		listener
 	);
 
+	// const API_KEY = import.meta.env.VITE_INFURA_API_KEY;
```

Comment:
What's that?

Reply:
a mistake... i was doing a test and left this commented...

Reply:
removed